### PR TITLE
test mic tests

### PR DESCRIPTION
**Pull request by tsvetomir-nedyalkov**

### CODE REVIEW

File: test/integration/tests/crud-entity/common-crud.test.js [hunk 1]
```diff
@@ -108,26 +114,45 @@ function testFunc() {
...
+
+                    var email = windows[0].document.getElementById('email')
+                    var pass = windows[0].document.getElementById('pass')
+                    var loginButton = windows[0].document.getElementById('loginbutton')
+                    if (email && pass && loginButton)
+                      {
+                        email.value="system.everlive@gmail.com"
+                        pass.value="LE9YIvKJc4L7dlOuH831"
+                        loginButton.click();
+                        console.log('clearrrrrrr');
```
Comment (thomasconner) [inline test/integration/tests/crud-entity/common-crud.test.js]: Should this `console.log()` be removed?

File: test/integration/tests/crud-entity/common-crud.test.js [hunk 2]
```diff
@@ -108,26 +114,45 @@ function testFunc() {
...
-                      }
-                      expect(result).to.deep.equal(dataStoreType === Kinvey.DataStoreType.Cache ? entity3 : undefined);
-                      done();
-                    })
-                    .catch(done);
-                } catch (error) {
-                  done(error);
-                }
-                return Promise.resolve();
+          it.only('should return all the entities', (done) => {
```
Comment (thomasconner) [inline test/integration/tests/crud-entity/common-crud.test.js]: It doesn't look like this is testing anything. It looks like it should be testing if all entities are returned.

Comment (tsvetomir-nedyalkov) [reply test/integration/tests/crud-entity/common-crud.test.js]: I have created this PR only for testing purposes - to test something in Travis and it is not for a review. I think, that I did not assign it for a review to anyone...

Comment (tsvetomir-nedyalkov) [reply test/integration/tests/crud-entity/common-crud.test.js]: Closing the PR - I will modify the travis.yml in my branch in order to test my changes in Travis.

**Pull request closed without merging**